InvenTree/report/templatetags/report.py

```diff
@@ -28,11 +28,15 @@ def asset(filename):
     Raises:
         FileNotFoundError if file does not exist
     """
+    if type(filename) is SafeString:
+        # Prepend an empty string to enforce 'stringiness'
+        filename = '' + filename
```

Comment:
question: wouldn't `str()` be a cleaner solution or is that not working?

Reply:
Nope, tried that, the addition of an empty string was called out on the django bug tracker as the "right" solution :p

Reply:
Interesting... does not look pythonic but if upstream thinks it is a good solution who am I to judge?